src/alire/alire-directories.adb

```diff
@@ -349,6 +350,22 @@ package body Alire.Directories is
          return;
       end if;
 
+      --  Force writability of folder when in Windows, as some tools (e.g. git)
+      --  that create read-only files will cause a Use_Error
+
+      if Kind (This.Filename) = Directory and then
+        GNAT.OS_Lib.Directory_Separator = '\'
```

Comment:
Is this to detect Windows? I think we have a better way:
```ada
GNATCOLL.OS.Constants.OS = GNATCOLL.OS.Windows
```

Reply:
Right you are